src/containers/pages/GenericJurisdictionReport/helpers.ts

```diff
@@ -117,6 +117,63 @@ export const ZambiaJurisdictionsColumns = [
   },
 ];
 
+/** columns for Zambia IRS jurisdictions districts and Catchment Area */
+export const ZambiaLowerJurisdictionsColumns = [
```

Comment:
Any reason why we are adding `ZambiaLowerJurisdictionsColumns` instead of modifying `ZambiaJurisdictionsColumns`?

Reply:
from the issue when displaying country and province levels the old table structure is maintained.